Rivers of London/rivers-of-london.html

```diff
@@ -575,6 +591,116 @@ <h3>Equipment</h3>
     </div>
   </section>
 
+  <!------------------------- Translations? ------------------------->
+  <input type="hidden" name="attr_str_txt" value="Strength"/>
+  <input type="hidden" name="attr_con_txt" value="Constitution"/>
+  <input type="hidden" name="attr_dex_txt" value="Dexerity"/>
+  <input type="hidden" name="attr_int_txt" value="Intelligence"/>
+  <input type="hidden" name="attr_pow_txt" value="Power"/>
+  <input type="hidden" name="attr_luck_txt" value="Luck"/>
+
+  <input type="hidden" name="attr_athletics_txt" value="Athletics"/>
+  <input type="hidden" name="attr_drive_txt" value="Drive"/>
+  <input type="hidden" name="attr_navigate_txt" value="Navigate"/>
+  <input type="hidden" name="attr_observation_txt" value="Observation"/>
+  <input type="hidden" name="attr_read_person_txt" value="Read Person"/>
+  <input type="hidden" name="attr_research_txt" value="Research"/>
+  <input type="hidden" name="attr_sense_vestigia_txt" value="Sense Vestigia"/>
+  <input type="hidden" name="attr_social_txt" value="Social"/>
+  <input type="hidden" name="attr_stealth_txt" value="Stealth"/>
+
+  <!--  Roll Template
+    --  &{template:rol} {{name=Test}} {{success=[[50]]}} {{hard=[[floor(50/2)]]}} {{roll1=[[1d100]]}}
+    -->
+  <rolltemplate class="sheet-rolltemplate-rol">
+    <table>
+      <caption>{{name}} ({{success}}/{{hard}})</caption>
+      <tr style="background:#BEBEBE;">
+        <td class="template_label" data-i18n="result">Result:</td>
+          <!-- 01 is always Critical -->
+          {{#rollTotal() roll1 1}} 
+            <td style="background:Lime;" class="template_value"><b data-i18n="critical">Critical</b></td>
+          {{/rollTotal() roll1 1}} 
+          <!-- Is Success -->
+          {{#rollBetween() roll1 2 success}}
+            <!-- Is Standard -->
+            {{#rollGreater() roll1 hard}}
+              <td style="background:DarkGreen;" class="template_value" data-i18n="success">Success</td>
```

Comment:
I'd rather handle the styling in the CSS. This would allow that:
```suggestion
              <div class="template_value hard-success" data-i18n="success">Success</div>
```

Right now, I find the results hard to read, and it would be easier to tweak in the CSS alone:
![image](https://user-images.githubusercontent.com/2582983/232238448-57fda9ba-ca1c-41a5-adc2-14fd2aa9d6c3.png)

Reply:
yea, that makes sense.